website/src/pages/index.js

```diff
@@ -21,16 +21,13 @@ function Hero() {
                     <div className={clsx(styles.relative, 'row')}>
                         <div className="col">
                             <h1 className={styles.tagline}>
-                                Crawlee is a web<br /> scraping and browser<br /> automation library
-                            </h1>
-                            <h1 className={styles.tagline}>
-                                Crawlee is a <span>web<br /> scraping</span> and <span>browser<br /> automation</span> library
+                                Build your <span>Python<br /> web crawlers</span><br /> using <span>Crawlee</span> 
```

Comment:
by removing the second h1 you disabled the gradient for the wrapped words, it looks quite ugly now
<img width="667" alt="image" src="https://github.com/user-attachments/assets/27a6a05c-0833-4a69-b23d-d0c6b4bee599">